### PR TITLE
Add a pull requests view similar to ocp-what-merged

### PR DESCRIPTION
**Pull request by stbenjam**
[TRT-816](https://issues.redhat.com//browse/TRT-816)

This add a "Pull Requests" page under Presubmits for viewing recently merged PR's, their job history, and the first release payload they appeared in.

image

Todo:
- [X] Link to first release payload
- [X] Show indicator for accepted/rejected payload
- [x] Fix filtering by date ranges

### CODE REVIEW

**Comment by stbenjam** [comment]
This is ready for a review but there's still a few things for me to clean up, and I need to fix the date filtering widget

/hold

File: pkg/api/pull_requests.go [hunk 1]
```diff
@@ -10,3 +10,32 @@ import (
...
 	return query.PullRequestReport(dbc, filterOpts, release)
 }
+
+type PullRequestPayloadHistory struct {
+	Release     string     `json:"release"`
+	Stream      string     `json:"stream"`
+	ReleaseTags [][]string `json:"release_tags"`
+}
+
+func GetPullRequestPayloadHistory(dbc *db.DB, pullRequestID int) ([]PullRequestPayloadHistory, error) {
```
Comment (DennisPeriquet) [inline pkg/api/pull_requests.go:20]: this doesn't seem to be called anywhere

File: sippy-ng/src/pull_requests/PullRequestsTable.js [hunk 2]
```diff
@@ -178,9 +265,25 @@ export default function PullRequestsTable(props) {
         )
       },
     },
+    history: {
+      field: 'history',
+      headerName: 'History',
```
Comment (DennisPeriquet) [inline sippy-ng/src/pull_requests/PullRequestsTable.js:270]: On this one and the GitHub, if you click the sort arrow, the output gets messed up.  Not sure how to exclude those from the `updateSortModel` (assuming that's how you do it).

Comment (stbenjam) [reply sippy-ng/src/pull_requests/PullRequestsTable.js:270]: It's just a matter of setting sortable to false for this column - https://mui.com/x/react-data-grid/sorting/#disable-the-sorting

File: sippy-ng/src/pull_requests/PullRequestsTable.js
Comment (DennisPeriquet) [inline sippy-ng/src/pull_requests/PullRequestsTable.js:1]: Using `Unmerged` view seems to show PRs that are merged: http://.../sippy-ng/pull_requests/Presubmits?sort=desc&sortField=merged_at&view=Unmerged

Comment (stbenjam) [reply sippy-ng/src/pull_requests/PullRequestsTable.js:1]: Hm, I'm not even sure where I intended to use that view, I'll remove it.  You can get this view by just filtering by merged_at is empty.

Other filters on merged_at don't work at the moment though (i.e. date ranges)

**Pull request merged**